modules/ROOT/pages/getting-started-libvirt.adoc

```diff
@@ -27,18 +27,24 @@ VCPUS="2"
 RAM_MB="2048"
 STREAM="stable"
 DISK_GB="10"
+For x86 / aarch64,
+IGNITION_DEVICE="qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=${IGNITION_CONFIG}""
+
+For s390x / ppc64le,
+IGNITION_DEVICE="disk path=${IGNITION_CONFIG},format=raw,readonly=on,serial=ignition,startup_policy=optional"
```

Comment:
Since this is simulating bash let's add a comment `#` character to the beginning of the comment. Also let's include the `--` in the variable here and change the name to `IGNITION_DEVICE_ARG`: 

```suggestion
# For x86 / aarch64,
IGNITION_DEVICE_ARG="--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=${IGNITION_CONFIG}""
# For s390x / ppc64le,
IGNITION_DEVICE_ARG="--disk path=${IGNITION_CONFIG},format=raw,readonly=on,serial=ignition,startup_policy=optional"
```

Then below you can do something like:

```
--network bridge=virbr0 $IGNITION_DEVICE_ARG
```

and let's make the same modifications in the qemu steps too.

Reply:
addressed in eab3b9da4c5293a514974465205b2a8bfe8e28e7